### DISCUSSION TITLE
Inner join is more slow than `ClickHouse`

### DISCUSSION THREAD

**Discussion by lgbo-ustc** [apache/doris#11593] [A - General / Q&A]
We have a table which has 62659998 rows. We run the following query in three nodes cluster both for Doris and ClickHouse
```SQL
select t1.day, t1.hour, count(1) from

(select * from test.server_event where day = '2022-08-07' and hour >= '21') as t1

join

(select * from test.server_event where day = '2022-08-07' and hour >= '21') as t2

on t1.rtime = t2.rtime and t1.msgId = t2.msgId and t1.rip = t2.rip

group by t1.day, t1.hour;
```

Doris takes more than 2min, but ClickHouse takes about 30s.
Is this result is expected? Or Do I miss some settings?

![image](https://user-images.githubusercontent.com/2599447/183397737-7adffb22-2644-455c-8f0b-d4c424bce5cb.png)

**Comment by dataalive** [comment]
which Doris version ?

**Reply by lgbo-ustc** [reply]
latest master branch
